app/operators/bill-plans-new.php

```diff
@@ -139,7 +139,7 @@
                         $format = "A new %s named %s has been successfully added to database. %d %s have been associated to this %s";
                         $successMsg = sprintf($format . ' [<a href="bill-plans-edit.php?planName=%s" title="Edit">Edit</a>]', t('all','PlanName'),
                                               $planName_enc, $groupsCount, t('title','Profiles'), t('all','PlanName'), urlencode($planName_enc));
-                        $logAction .= sprintf("$format on page: ", t('all','PlanName'), $groupsCount, t('title','Profiles'), t('all','PlanName'));
+                        $logAction .= sprintf("$format on page: ", t('all','PlanName'), $groupsCount, t('title','Profiles'), t('all','PlanName'), '');
```

Comment:
Actually, it's rightfully missing an element because it expects 5 to complete the variables in the `$format` string. 

We can fix it like this:

```suggestion
                        $logAction .= sprintf("$format on page: ", t('all','PlanName'), $planName, $groupsCount, t('title','Profiles'), t('all','PlanName'));
```